backend/scraper/data_scrapers/counted/scraper_data/README.txt

```diff
@@ -0,0 +1,3 @@
+Please credit the Guardian as the source of this data in any work you do with it. Please link to http://www.theguardian.com/thecounted.
```

Comment:
There should be a set of fields that indicate the source of this data as outlined in the [Incident Model](https://docs.google.com/document/d/1PCsRIKr0PF14Ul_qnem_UrUHz5RA0-5EmP5NIbDDQGA/edit?usp=sharing). Specifically:

- Publication Name
- Publication Date
- Author
- URL (Optional)

Let's try to get that information added with this.